skaffold/skaffold_vizier.yaml

```diff
@@ -138,9 +138,15 @@ profiles:
     path: /manifests/kustomize/paths
     value:
     - k8s/vizier/persistent_metadata/aarch64
+# Note: You will want to stick with a sysroot based build (-p x86_64_sysroot or -p aarch64_sysroot),
+# but you may want to change the --complication_mode setting based on your needs.
+# opt builds remove assert/debug checks, while dbg builds work with debuggers (gdb).
+# See the bazel docs for more details https://bazel.build/docs/user-manual#compilation-mode
 - name: x86_64_sysroot
   patches:
   - op: add
     path: /build/artifacts/context=./bazel/args
     value:
     - --config=x86_64_sysroot
+    - --compilation_mode=dbg
+#    - --compilation_mode=opt
```

Comment:
Since this is already in the DEVELOPMENT.md, I think it would best to omit any changes to this file. The comments about compilation_mode go for any skaffold file in the repo (of which there are a few), so I think the DEVELOPMENT.md location covers our bases.

Reply:
reverted to main